packages/devextreme/testing/helpers/device.js

```diff
@@ -0,0 +1,16 @@
+import devices from '__internal/core/m_devices';
+
+export const shouldSkipOnDevice = ({
```

Comment:
let's use plural to emphasize array should be passed
```suggestion
export const shouldSkipOnDevices = ({
```